### PR TITLE
Azure Functions - Support Linux consumption app deployment

### PR DESCRIPTION
**Pull request by vincentdass**

### CODE REVIEW

File: Tasks/AzureFunctionAppContainerV1/task.loc.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 3
```
Comment (SumiranAgg) [inline Tasks/AzureFunctionAppContainerV1/task.loc.json]: check patch versions. There have been multiple other PRs on these tasks now.

File: Tasks/AzureFunctionAppV1/Strings/resources.resjson/en-US/resources.resjson [hunk 2]
```diff
@@ -178,5 +179,10 @@
   "loc.messages.JarPathNotPresent": "Java jar path is not present",
   "loc.messages.FailedToUpdateApplicationInsightsResource": "Failed to update Application Insights '%s' Resource. Error: %s",
   "loc.messages.InvalidDockerImageName": "Invalid Docker hub image name provided.",
-  "loc.messages.MsBuildPackageNotSupported": "Deployment of msBuild generated package is not supported. Change package format or use Azure App Service Deploy task."
+  "loc.messages.MsBuildPackageNotSupported": "Deployment of msBuild generated package is not supported. Change package format or use Azure App Service Deploy task.",
+  "loc.messages.FailedToGetStorageAccountDetails": "Unable to find the storage account associated with the function app.",
+  "loc.messages.FailedToSyncTriggers": "Failed to sync triggers '%s' application settings. Error: %s",
```
Comment (SumiranAgg) [inline Tasks/AzureFunctionAppV1/Strings/resources.resjson/en-US/resources.resjson]: What does this message mean? "Application settings" seems to be disconnected in the sentence.

Comment (vincentdass) [reply Tasks/AzureFunctionAppV1/Strings/resources.resjson/en-US/resources.resjson]: @20shivangi , Can you please rephrase the sentence?
`Failed to sync triggers for web app '%s'. Error: %s`

File: Tasks/Common/AzureRmDeploy-common/azure-arm-rest/azure-arm-app-service.ts [hunk 3]
```diff
@@ -209,7 +207,7 @@ export class AzureAppService {
         var isNewValueUpdated: boolean = false;
         for(var key in addProperties) {
             if(applicationSettings.properties[key] != addProperties[key]) {
-                tl.debug(`old value : ${applicationSettings.properties[key]}. new value: ${addProperties[key]}`);
+                tl.debug(`Value of ${key} has been changed`);
```
Comment (SumiranAgg) [inline Tasks/Common/AzureRmDeploy-common/azure-arm-rest/azure-arm-app-service.ts]: Print updated value. Might be relevant info for customer.

Comment (vincentdass) [reply Tasks/Common/AzureRmDeploy-common/azure-arm-rest/azure-arm-app-service.ts]: `Value of ${key} has been changed to ${addProperties[key]}`

Comment (20shivangi) [reply Tasks/Common/AzureRmDeploy-common/azure-arm-rest/azure-arm-app-service.ts]: It is done.

File: Tasks/AzureFunctionAppV1/Strings/resources.resjson/en-US/resources.resjson (hunk 2 above)
Comment (20shivangi) [reply Tasks/AzureFunctionAppV1/Strings/resources.resjson/en-US/resources.resjson]: It is done.

**Pull request merged**